lib/routemap_cli.c

```diff
@@ -132,47 +132,6 @@ void route_map_instance_show(struct vty *vty, struct lyd_node *dnode,
 		 */
 		return;
 	}
-
```

Comment:
`rmi = nb_running_get_entry(dnode, NULL, false);` above should be also removed.

Reply:
Done

Reply:
Hi @idryzhov,
Thanks for your comments.
I have addressed these and updated the patch